backend/services/implementations/testSessionService.ts

```diff
@@ -104,19 +108,21 @@ class TestSessionService implements ITestSessionService {
     accessCode: string,
   ): Promise<TestSessionResponseDTO> {
     let testSessionDtos: Array<TestSessionResponseDTO> = [];
-
+    const currentDate = new Date();
     try {
       const testSessions: Array<TestSession> = await MgTestSession.find({
         accessCode: { $eq: accessCode },
+        startDate: { $lte: currentDate },
+        endDate: { $gte: currentDate },
```

Comment:
side note, we should prob make explicit to user somewhere that dates are inclusive (if its not in designs)

Reply:
yup. I think we can ask design to add it in the copy when creating the form